src/translations/en-US.json

```diff
@@ -12,12 +12,15 @@
   "h3" : "Punishment",
   "home" : "Home",
   "about" : "About",
+  "whattodo": "What To Do",
```

Comment:
```suggestion
  "whatTodo.title": "What To Do",
```

nit: We should name the title string as such, so that it is easier to read what that is for so that as this file get larger, it would be easier to updated if needed